packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -36,43 +53,42 @@ interface LiquidityResults {
   [liquidity: number]: number;
 }
 
-// because flexsearch is a UMD type lib
-import FlexSearch = require("flexsearch");
-
 /**
  * Market specific derived DB intended for filtering purposes
  */
 export class MarketDB extends DerivedDB {
   protected augur: Augur;
   private readonly events = new Subscriptions(augurEmitter);
-  private flexSearch?: FlexSearch;
+  private flexSearchIndex?: Index<MarketFields>;
   readonly liquiditySpreads = [10, 15, 20, 100];
 
   constructor(db: DB, networkId: number, augur: Augur) {
     super(db, networkId, "Markets", ["MarketCreated", "MarketVolumeChanged", "MarketOIChanged"], ["market"]);
-    
+
     this.augur = augur;
 
     this.events.subscribe('DerivedDB:updated:CurrentOrders', this.syncOrderBooks);
 
-    this.flexSearch = new FlexSearch({
-      doc: {
-        id: "id",
-        start: "start",
-        end: "end",
-        field: [
-          "market",
-          "category1",
-          "category2",
-          "category3",
-          "description",
-          "longDescription",
-          "resolutionSource",
-          "_scalarDenomination",
-          "tags",
-        ],
-      },
-    });
+    this.flexSearchIndex = flexSearch.create(
+      {
+        doc: {
+          id: "id",
+          start: "start",
+          end: "end",
+          field: [
+            "market",
+            "category1",
+            "category2",
+            "category3",
+            "description",
+            "longDescription",
+            "resolutionSource",
+            "_scalarDenomination",
+            "tags",
```

Comment:
Tags should not exist now. Also can we add market creater

Reply:
Good catch! I'll remove the tags and create another task for making marketCreator full-text-searchable.